aiida/cmdline/commands/cmd_group.py

```diff
@@ -84,6 +84,70 @@ def group_remove_nodes(group, nodes, clear, force):
         group.remove_nodes(nodes)
 
 
+@verdi_group.command('move-nodes')
+@options.SOURCE_GROUP(required=True)
+@options.TARGET_GROUP(required=True)
+@arguments.NODES()
+@options.FORCE(help='Do not ask for confirmation and skip all checks.')
+@with_dbenv()
+def group_move_nodes(source_group, target_group, nodes, force):
+    """Move nodes from one group to another."""
+    from aiida.orm import Group, Node, QueryBuilder
+
+    source_label = source_group.label
+    source_class = source_group.__class__.__name__
+
+    target_label = target_group.label
+    target_class = target_group.__class__.__name__
+
+    if not force:
+
+        if source_label == target_label:
+            echo.echo_critical(f'Source and target group are the same: {source_class}<{source_label}>.')
+
+        if nodes:
+            node_pks = [node.pk for node in nodes]
+
+            query = QueryBuilder()
+            query.append(Group, filters={'label': source_label}, tag='group')
+            query.append(Node, with_group='group', filters={'id': {'in': node_pks}}, project='id')
+
+            source_group_node_pks = query.all(flat=True)
+
+            if not source_group_node_pks:
+                echo.echo_critical(f'None of the specified nodes are in {source_class}<{source_label}>.')
```

Comment:
Your formatting is exactly that of `Group.__str__` so you can just plonk it in. Apply also in other places
```suggestion
                echo.echo_critical(f'None of the specified nodes are in {source_group}.')
```

Reply:
Right, I probably should have reviewed this old PR myself before highlighting it at the meeting. 😅